packages/example/src/pages/guides/configuration.mdx

```diff
@@ -259,9 +259,8 @@ plugins: [
 
 - `mdxExtensions` – change the file extensions processed by `gatsby-mdx`
   (default ['.mdx', '.md']).
-- `pngCompressionSpeed` - a speed/quality trade-off from 1 (brute-force) to 10
```

Comment:
This option doesn't work any more, removing the documentation ahead of actually removing it in the next major.